src/modm/platform/clock/stm32/rcc.hpp.in

```diff
@@ -172,6 +201,7 @@ public:
 		Div128 = RCC_{{cfgr1}}_HPRE_DIV128,
 		Div256 = RCC_{{cfgr1}}_HPRE_DIV256,
 		Div512 = RCC_{{cfgr1}}_HPRE_DIV512
+%% endif
```

Comment:
ST, why? :crying_cat_face:

Reply:
The L5 headers have none of the `RCC_CFGR_HPRE_DIV4` and similar defines like all other STM32 headers ...

Reply:
I opened an issue about this and they won't fix it, since these definitions have been moved to the Cube LL headers.
So… dunno what to do about this, except patch them back in manually in our `device.hpp` header, which is dumb.

Reply:
Found it: https://github.com/STMicroelectronics/STM32CubeG0/issues/3#issuecomment-506258241

Reply:
Has this been resolved?